content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -179,6 +180,11 @@ export class _DiscoveryStreamBase extends React.PureComponent {
       return this.renderLayout(layoutRender);
     }
 
+    // Send SPOCS Fill if any.
+    if (spocsFill.length) {
+      this.props.dispatch(ac.DiscoveryStreamSpocsFill({spoc_fills: spocsFill}));
```

Comment:
Is this being after:
```
    if (!config.collapsible) {
      return this.renderLayout(layoutRender);
    }
```

An issue?

Reply:
Do you need to make sure this only gets called on initial render, or if something changes that causes the page to update and re render, does this get fired again?

example, if you block a spoc or rec from the story context menu, it causes the page to call render again.